src/libstd/sys/windows/handle.rs

```diff
@@ -106,3 +107,13 @@ impl RawHandle {
         Ok(Handle::new(ret))
     }
 }
+
+impl PartialEq<RawHandle> for RawHandle {
```

Comment:
I believe the parameter on `PartialEq` is defaulted to `Self` so this and the ones around I believe can be left off.